apps/ensadmin/.env.local.example

```diff
@@ -6,9 +6,12 @@
 # the default public URL will be based on localhost and PORT.
 ENSADMIN_PUBLIC_URL=http://localhost:4173
 
-# Preferred ENSNode URL (used by default when ENSAdmin is opened)
-# If not set, the fallback URL from `PREFERRED_ENSNODE_URL` const is used.
-NEXT_PUBLIC_PREFERRED_ENSNODE_URL=https://alpha.ensnode.io
+# Default ENSNode URLs (used by default when ENSAdmin is opened)
+# If not set, the fallback URL from `DEFAULT_ENSNODE_URL` const is used.
+# Note: it must be a comma-separated list of URLs that are accessible from web browser
+# (i.e. it cannot be a hostname in docker network)
+# Note: the first URL will be used as default ENSNode connection URL
+NEXT_PUBLIC_DEFAULT_ENSNODE_URLS=https://api.alpha.ensnode.io,https://api.mainnet.ensnode.io,https://api.sepolia.ensnode.io,https://api.holesky.ensnode.io
```

Comment:
This change addresses the following requirement from PR feedback:
> X is the ordered list of ENSNode connections that are hardcoded into ENSAdmin

Connections hardcoded on the environment configuration level.